modules/extras.py

```diff
@@ -132,6 +132,7 @@ def filename_nothing():
     tertiary_model_info = sd_models.checkpoints_list[tertiary_model_name] if theta_func1 else None
 
     result_is_inpainting_model = False
+    result_is_pix2pix_model = False

```

Comment:
For clarity, I'd suggest using the full name "instruct_pix2pix" instead of just "pix2pix".

Reply:
Ok, just pushed the changes which renames this variable to the more precise name.